libp2p/Service.cpp

```diff
@@ -1193,20 +1195,30 @@ bool Service::updatePeersToIni(std::map<dev::network::NodeIPEndpoint, NodeID> co
         auto pos1 = fileData.find("p2p");
         if (pos1 == fileData.npos)
         {
+            SERVICE_LOG(DEBUG) << LOG_DESC("fail to locate peers' information in config file");
             return false;
         }
         auto pos2 = fileData.find("node.0=", pos1);
         auto pos3 = fileData.find("certificate_blacklist");
         if (pos2 == fileData.npos || pos3 == fileData.npos || pos2 >= pos3)
         {
+            SERVICE_LOG(DEBUG) << LOG_DESC("fail to locate peers' information in config file");
             return false;
         }
         fileData.replace(pos2, pos3 - pos2 - 1, tmpdata);
```

Comment:
the method to update nodeX fields is not appropriate， if the format of config.ini changed, eg: some other config 
items add before  certificate_blacklist, the replace operation will overide all the config items between p2p and certificate_blacklist

Reply:
got it